_script/readme_generator_samples/help

```diff
@@ -4,8 +4,9 @@ set -euo pipefail
 cd $(dirname $0)/../..
 
 cd out/
-cmdline1="./tcpgoon --help"
-cmdline2="./tcpgoon run --help"
+PATH=".:$PATH"
```

Comment:
is there any rational behind this change?

Reply:
yes: people installing tcpgoon will get it installed probably in their path, plus we are not providing compilation instructions. So that's why "./" is being removed from the README, as it only works if you build and jump into the directory with the binary..